htdocs/langs/ja_JP/propal.lang

```diff
@@ -85,3 +85,7 @@ ProposalCustomerSignature=承諾書、会社印、日付、署名
 ProposalsStatisticsSuppliers=Vendor proposals statistics
 CaseFollowedBy=Case followed by
 SignedOnly=Signed only
+IdProposal=提案番号
```

Comment:
There is no need to update language file. They will be erased.
Just update the english en_US file. And wait. Translation will be done by translators on transifex.